api/src/Page/Shipment.php

```diff
@@ -1217,7 +1218,7 @@ function _dispatch_dewar()
                         $error_json = json_decode($e->getMessage());
                         $error_response = $error_json->content->detail ?? $e->getMessage();
                         $error_status = $error_json->status ? $error_json->status : 400; // Status can be 0
-                        $this->_error("Shipping service error: $error_response", $error_status);
+                        $this->_error("Shipping service error: " . json_encode($error_response), $error_status);
```

Comment:
Is this change required? If it exists, `$error_json->content->detail` should be a string, if not `$error_reponse` will be whatever string is returned by `$e->getMessage()`.

Reply:
This is what I get back from `$error_json->content->detail` if I run it from a local machine and forget to put .diamond.ac.uk:
```
[{"loc":["body","origin_url"],"msg":"URL host invalid, top level domain required","type":"value_error.url.host"}]
```

Reply:
Ah, fair enough. I forgot that validation errors didn't follow the same convention.